ios/PolyPodApp/Core/PolyPodCore.swift

```diff
@@ -14,7 +14,7 @@ final class PolyPodCore {
         let path = try! FileManager.default.url(for: .documentDirectory, in: .userDomainMask, appropriateFor: nil, create: false).path
         containerPath = NSString(string: path).utf8String!
         
-        corePointer = new_core(.init(tag: .init(0), .init(.init(default_: containerPath))))
+        corePointer = new_core(.default(containerPath))
```

Comment:
This is the ideal minimal setup needed for core. When from iOS we just tell Core where it can store its files. And core performs all of the operations internally.

This is the state based on https://github.com/polypoly-eu/polyPod-platform-core-experiments/pull/11/commits/7c8526bbc3ccbea89d86d82b8185075d06517577